Common/Data/Fundamental/Generated/CompanyReference.cs

```diff
@@ -0,0 +1,277 @@
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+//------------------------------------------------------------------------------
+// This code was generated by a tool.
+// Changes to this file may cause incorrect behavior and will be lost if
+// the code is regenerated.
+//------------------------------------------------------------------------------
+
+// To get up to date fundamental definition files for your hedgefund contact sales@quantconnect.com
+
+using System;
+using Newtonsoft.Json;
+
+namespace QuantConnect.Data.Fundamental
+{
+	/// <summary>
+	/// Definition of the CompanyReference class
+	/// </summary>
+	public class CompanyReference : BaseData
+	{
+		/// <summary>
+		/// 10-digit unique and unchanging Morningstar identifier assigned to every company.
+		/// </summary>
+		/// <remarks>
+		/// Morningstar DataId: 1
+		/// </remarks>
+		[JsonProperty("1")]
+		public string CompanyId { get; set; }
+
+		/// <summary>
+		/// 25-character max abbreviated name of the firm.  In most cases, the short name will simply be the Legal Name less the
+		/// “Corporation”, “Corp.”, “Inc.”, “Incorporated”, etc…
+		/// </summary>
+		/// <remarks>
+		/// Morningstar DataId: 2
+		/// </remarks>
+		[JsonProperty("2")]
+		public string ShortName { get; set; }
+
+		/// <summary>
+		/// The English translation of the foreign legal name if/when applicable.
+		/// </summary>
+		/// <remarks>
+		/// Morningstar DataId: 3
+		/// </remarks>
+		[JsonProperty("3")]
+		public string StandardName { get; set; }
+
+		/// <summary>
+		/// The full name of the registrant as specified in its charter, and most often found on the front cover of the 10K/10Q/20F filing.
+		/// </summary>
+		/// <remarks>
+		/// Morningstar DataId: 4
+		/// </remarks>
+		[JsonProperty("4")]
+		public string LegalName { get; set; }
+
+		/// <summary>
+		/// 3 Character ISO code of the country where the firm is domiciled. See separate reference document for Country Mappings.
+		/// </summary>
+		/// <remarks>
+		/// Morningstar DataId: 5
+		/// </remarks>
+		[JsonProperty("5")]
+		public string CountryId { get; set; }
+
+		/// <summary>
+		/// The Central Index Key; a corporate identifier assigned by the Securities and Exchange Commission (SEC).
+		/// </summary>
+		/// <remarks>
+		/// Morningstar DataId: 6
+		/// </remarks>
+		[JsonProperty("6")]
+		public string CIK { get; set; }
+
+		/// <summary>
+		/// At the Company level; each company is assigned to 1 of 3 possible status classifications; (U) Public, (V) Private, or (O) Obsolete:
+		///  Public-Firm is operating and currently has at least one common share class that is currently trading on a public exchange.
```

Comment:
I wonder if this character will cause issues with readers / XML parsers? Can you please strip it out and replace all non A-Z,0-9 characters with safe versions? E.g. there are a few " ’ " instead of " ' " as well as " “1”  "  instead of "  "1" "